app/models/plate_appearance.rb

```diff
@@ -242,4 +119,68 @@ def merge_possible_outcomes
       raise "unreachable condition"
     end
   end
+
+  def simple_out?(code)
+    simple_outs = ["strikeout", /^popout/, /^lineout/, /^foulout/, /^lo/, /^CATCH/]
+    simple_outs.each do |simple_out|
+      if simple_out.class == Regexp
+        return true if code =~ simple_out
+      else
+        return true if code == simple_out
+      end
+    end
+    false
+  end
+
+  def home_run?(code)
+    %w(HOMERUN HR N-HR).include?(code)
+  end
+
+  def single_with_two_asterisk?(code)
+    code =~ /^SI/i && code =~ /\*{2}$/
+  end
+
+  def double?(code)
+    code =~ /^DO/i
+  end
+
+  def triple?(code)
+    code =~ /^TR/i
+  end
+
+  def simple_single_like_event?(code)
+    (code =~ /^SI/i && code !~ /\*{2}$/) || code =~ /^HBP/ || code =~ /^WALK/
+  end
+
+  def groundball_a?(code)
+    code.downcase.first(2) == "gb" && code =~ /A\+?$/
+  end
+
+  def groundball_b?(code)
+    code.downcase.first(2) == "gb" && code =~ /B\+?$/
+  end
+
+  def groundball_c?(code)
+    code.downcase.first(2) == "gb" && code =~ /C$/
+  end
+
+  def groundball_x?(code)
+    code.downcase.first(2) == "gb" && code =~ /X\+?$/
+  end
+
+  def flyball_a?(code)
+    code.downcase.first(3) == "fly" && code =~ /A\.?$/
+  end
+
+  def flyball_b?(code)
+    code.downcase.first(3) == "fly" && code =~ /B.?$/
+  end
+
+  def flyball_c?(code)
+    code.downcase.first(3) == "fly" && code =~ /C\.?$/
```

Comment:
Same here. You shouldn't have to escape wildcard characters in Ruby's regex engine.